### PR TITLE
School aggregate

### PR DESCRIPTION
**Pull request by jaduff**
Fixed event handler logic to work with both tests and LabLog.
Single ReplaySchoolEvent method can be used to update ReadModel or to RebuildReadModel in the event of consistency failure or updating if versions behind.

### CODE REVIEW

File: src/LabLog.Domain/Entities/Computer.cs [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
     {
-        public int ComputerId { get; }
         public string ComputerName { get; }
+        public Guid RoomId {get;}
+        public string SerialNumber {get;}
+        public int Position {get;}
 
-        public Computer(int computerId,
-            string computerName)
+        public Computer(Guid roomId, string serialNumber,
```
Comment (simonjduff) [inline src/LabLog.Domain/Entities/Computer.cs]: Why does a computer care what room it's in?

File: src/LabLog/Controllers/AdminController.cs [hunk 2]
```diff
@@ -54,8 +51,8 @@ public IActionResult RebuildReadModel()
                     .OrderBy(o => (o.Version));
                 _school.ReplaySchoolEvents(schoolEvents);
```
Comment (simonjduff) [inline src/LabLog/Controllers/AdminController.cs:52]: I can't see where schoolEvents is defined. Is it class level? Should it start with an underscore?

Comment (jaduff) [reply src/LabLog/Controllers/AdminController.cs:52]: It is defined just before:
```
var schoolEvents = _db.LabEvents
                    .Where(w => (w.SchoolId == e.SchoolId))
                    .OrderBy(o => (o.Version));
_school.ReplaySchoolEvents(schoolEvents);
```
It is a local variable, but not a protected one. What is the convention regarding underscores in variable names?

Comment (simonjduff) [reply src/LabLog/Controllers/AdminController.cs:52]: That's what I get for reviewing a PR on my phone :P

**Review by simonjduff — APPROVED** [review]

**Pull request merged**